doc/how_to_guides/moveit_launch_files/moveit_launch_files_tutorial.rst

```diff
@@ -213,3 +248,23 @@ Finally, we can tell our launch file to actually launch everything we described
                 hand_controller_spawner,
             ]
         )
+
+Launching a custom .cpp file
+----------------------------
+
+While not part of the Getting Started tutorial, another common node to launch is one that executes a custom .cpp file:
+
+.. code-block:: python
+
+    moveit_cpp_node = Node(
+        name="custom_program",
+        package="custom_program_package",
+        executable="custom_program",
+        output="screen",
+        parameters=[your,
+                    parameters,
+                    here],
```

Comment:
I wouldn't consider this too helpful because I always forget what the actual format of the parameters is -- it's a list of dictionaries or something? So I may consider using a real, valid input here instead of placeholder text.